angular-mentoring/src/app/coursesPage/courses/courses.component.html

```diff
@@ -0,0 +1,32 @@
+<section class="courses-section" *ngFor="let course of courses; let i = index;">
```

Comment:
Course item should be extracted to the separate component

Reply:
I separated course component.

Reply:
There should be 3 components: CoursePage, Courses, Course. Naming can be different